src/components/editor/editor-component.ts

```diff
@@ -287,10 +288,19 @@ export function view(
         updateMetadata,
         deleteNode,
       ),
-
       html.div(
         {
           id: 'editor',
+          onclick: (ev: Event) => {
+            if ((ev.target as El).tagName !== 'A') {
+              return
+            }
+            ev.preventDefault()
+            const link = (ev.target as El).getAttribute('href')
+            if (link !== null) {
+              selectNode(parseInt(link.substring(5), 10))
```

Comment:
you need to keep in mind a link maybe a regular http one. I suggest regex match against only the "note://\d" pattern

Reply:
Done.